api/file.go

```diff
@@ -246,8 +249,21 @@ func handleImages(filenames []string, fileData [][]byte, teamId, channelId, user
 			// Create preview
 			go func() {
 				var preview image.Image
-				if width > int(utils.Cfg.FileSettings.PreviewWidth) {
-					preview = imaging.Resize(img, utils.Cfg.FileSettings.PreviewWidth, utils.Cfg.FileSettings.PreviewHeight, imaging.Lanczos)
+				ratio := float64(height) / float64(width)
```

Comment:
Are you sure he didn't mean resizing the `<img>` object programatically? Changing the size of the actual image doesn't seem like a good solution in the long run because it won't affect any existing images at all and if the UI changes, then we'll need to update serverside code